src/content/guides/development.md

```diff
@@ -21,7 +21,7 @@ If you've been following the guides, you should have a solid understanding of so
 
 W> The tools in this guide are __only meant for development__, please __avoid__ using them in production!
 
-Before proceeding lets first set [`mode` to `'development'`](/configuration/mode/#mode-development).
+Before proceeding, let's first set [`mode` to `'development'`](/configuration/mode/#mode-development).
```

Comment:
```suggestion
Let's start from setting [`mode` to `'development'`](/configuration/mode/#mode-development).
```

Reply:
Maybe "Let's start **by** setting"? Sounds better...

Reply:
yup, sounds better, please go ahead and change